clients/extension/src/inpage/providers/xdefiKeplr.ts

```
@@ -29,11 +30,12 @@ import { CosmosMsgType, RequestMethod } from '../../utils/constants'
 import { getCosmosChainFromAddress } from '../../utils/cosmos/getCosmosChainFromAddress'
 import {
   Messaging,
-  SendTransactionResponse,
   TransactionDetails,
   TransactionType,
 } from '../../utils/interfaces'
 import { Cosmos } from './cosmos'
+import { shouldBeDefined } from '@lib/utils/assert/shouldBeDefined'
```

ESLint

[error] 'shouldBeDefined' is defined but never used.

(@typescript-eslint/no-unused-vars)